Update French.cs

neontry opened the pull request:
Spelling checked, missing lines completed, punctuation checked, replaced a few words to fit more, Replaced a line to its original meaning in the English file (the Unity version one). Feel free to double check and edit this request if needed.

NicoKuroKusagi commented:
Hello, the merge request is globaly good but there are still some points that need to be addressed before being validated:

- "upload" was translated into its french form "mettre en ligne" for every occurences, but "trigger" hasn't. To maintain
  consistency, every other occurences should be translated.

- "override" was translated with proper french expression but in this merge request got rolled back to the english word "override". 
  As stated before the whole translation needs consistency.

- for the section ABI_UI_BUILDPANEL_AVATAR_WARNING_VIEWPOINT, the provided translation is "la vue de cet avatar est défini" 
  but should be "la vue de cet avatar est définie"

neontry commented:
This should take care of the points mentioned above:
- Replaced the missedoccurrence "trigger" by its french form "déclencheur".
- Reverted "override" to its original translation "surcharge"
- Corrected the grammar mistake.
